data/src/main/java/com/jamieadkins/gwent/data/card/mapper/GwentCardMapper.kt

```diff
@@ -71,6 +71,14 @@ class GwentCardMapper @Inject constructor(
         }
     }
 
+    private fun cardSetIdToCardSet(cardSet: String): GwentCardSet {
+        return when (cardSet) {
+            CardSet.BASE_SET -> GwentCardSet.BASE
+            CardSet.THRONEBREAKER -> GwentCardSet.THRONEBREAKER
+            else -> throw Exception("Card set not found")
```

Comment:
If you merge with master, you'll see I stopped throwing an exception here. This is because when the DB is auto migrated this field will be empty, so every card will fail the map. I've changed it to just log the error and return a default value. The next time the user updates, the field will be populated properly. The user is also prompted to update every time the DB scheme changes.